code/modules/racial_roundstartperks/racial_perk.dm

```diff
@@ -0,0 +1,253 @@
+	desc = "You force your body to begin the process of removing toxins from your blood. All toxins, addictions, and stimulants are slowly purged while any toxin damage to your liver or body is healed but the effect leaves you exhausted.... and aswell making your body way more vunerable to toxins."
+	active = FALSE
+	passivePerk = FALSE
+
+/datum/perk/purgemaluses/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your body aches with the pain of its recent purge, you'll need more rest before using this effect again."))
+		return FALSE
+	cooldown_time = world.time + 15 MINUTES
+	user.visible_message("[user] shivers slightly as they begin to slow down.", "You start to feel quite chilly and tired as your body begins purging toxins within your blood.")
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("sataricillin", 5)
+	return ..()
+
+/datum/perk/purgemaluses/assign(mob/living/carbon/human/H)
+	..()
+	holder.toxin_mod_perk += 0.1 //Might be to high...
+
+/datum/perk/purgemaluses/remove()
+	holder.toxin_mod_perk -= 0.1
+	..()
+
+
+/datum/reagent/medicine/spaceacillin/sataricillin
+	name = "Sataricillin"
+	id = "sataricillin"
+	description = "An all-purpose antiviral agent naturally produced by several species that functions identically to Spaceacillin."
+	constant_metabolism = TRUE
+
+
+/datum/perk/viruspurge
+	name = "Pathogenical Response"
+	desc = "Your body is adept not only at curing toxins and regulating its blood flow but also fighting off infections and disease in any form. All infections within you are slowly cured and diseases progression slowed if not outright cured, similar to as if you were injected with spaceacillin. Severe infections or late stage diseases may still need additional medical aid and this cannot restore necrotic tissue. It uses the method of heating the body commonly refered as fever. Aswell making you extremely vunerable to toxins."
+	active = FALSE
+	passivePerk = FALSE
+
+
+/datum/perk/viruspurge/assign(mob/living/carbon/human/H)
+	..()
+	holder.toxin_mod_perk += 0.2 //Might be to high...
+
+/datum/perk/viruspurge/remove()
+	holder.toxin_mod_perk -= 0.2
+	..()
+
+
+/datum/perk/viruspurge/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your chemical sacks have not refilled yet, you'll need more rest before using this effect again."))
+		return FALSE
+	cooldown_time = world.time + 60 MINUTES
+	user.visible_message("[user] shivers slightly before taking a deep breath.... they look hot... are they in a heavy fever?", "You shiver slightly and take a deep breath before willing your bodies chemical sacks to open and begin purging infections.")
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("citokinesimol", 5)
+	return ..()
+
+/datum/reagent/medicine/citokinesimol
```

Comment:
This...chem is doing the exact same stuff I suggested on the earlier chem, making these two perks pretty much the same with almost no difference, **why**?